test/test_scc.jl

```diff
@@ -10,23 +9,29 @@ uds = [undiscount_scc(m, 2020), undiscount_scc(m, 2040), undiscount_scc(m, 2075)
 uds_compare = [0.8765, 0.5279, 0.2162] # From Excel PAGE-ICE
 @test uds ≈ uds_compare rtol=1e-4
 
-function checksccs(sccs, sccomp)
+function checksccs(sccs, scccomp)
     sccobs = [quantile(sccs, [.05, .25, .5, .75, .95]); mean(sccs)]
     sccerror = abs.((sccobs - scccomp) ./ [10, 1, 1, 1, 10, 1])
-    sccsd = std(sccs) / sqrt(1000)
+    sccsd = std(sccs) / sqrt(100_000)
     @test all(sccerror .< 3*sccsd) # average off by < 3x SD
 end
 
 m = getpage("RCP4.5 & SSP2", true, true, true)
-sccs = compute_scc_mcs(m, 1000, year=2020)
-checksccs(sccs, [37.031, 89.742, 161.868, 299.439, 847.020, 261.739])
+sccs = compute_scc_mcs(m, 100_000, year=2020)
+# TODO - get the baseline values to test against!
```

Comment:
@jrising I don't think this file has worked in a while, given there was a bug in the syntax and it probably was holding old PAGE2009 values but I'm really not sure?  As of now with a n of 1000 the tests weren't passing even on master so I'm thinking I should re-run with larger n and remake these tests?  Do you use these regularly currently?